coding/cpp_style_guide.rst

```diff
@@ -1242,6 +1242,36 @@ A class or struct definition must explicitly declare the privacy qualifier of it
 Although C++ provides the above definitions as defaults, some compilers generate warnings if explicit privacy qualifiers are not specified.
 This Rule will reduce unnecessary compiler warnings.
 
+.. _style-guide-cpp-5-2c:
+
+5-2c. Classes SHOULD explicitly declare or delete compiler-generated members
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All new C++ classes SHOULD explicitly default or delete all constructors and assignment operators that may be synthesized by the compiler.
+Similarly, destructors should be explicitly defined using the ``default`` keyword as well when no custom implementation is needed.
+In practice, that means all classes should explicitly define the follow six members using either ``default``, ``delete``, or a custom implementation (shown here defaulted):
```

Comment:
"follow" -> "following"

Reply:
I also see that it should be "five", not "six" (at one point I had a defaulted default constructor, too, but I remove that as I didn't want to imply that all classes should have a default constructor).